code/datums/votes/custom_vote.dm

```diff
@@ -1,9 +1,14 @@
 /// The max amount of options someone can have in a custom vote.
 #define MAX_CUSTOM_VOTE_OPTIONS 10
 
-/datum/vote/custom_vote
-	name = "Custom"
-	message = "Click here to start a custom vote."
+/datum/vote/custom_vote/fptp
+	name = "Custom FPTP"
+	message = "Click here to start a custom FPTP vote, FPTP (one selection per voter)"
```

Comment:
I think we should probably keep this intuitive as "simple vote" / "multi-vote". It's significantly easier for everyone to understand. FPTP and AV are way too jargon-y for me.

Reply:
okay